src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -230,6 +230,33 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
         }
 
+        /// <summary>
+        /// Gets an &lt;input&gt; elements "type" attribute value for the given <paramref name="modelExplorer"/>.
```

Comment:
Curious, where does the InputType calculation come into play within this method?